plugins/woocommerce/src/Internal/ComingSoon/ComingSoonAdminBarBadge.php

```diff
@@ -30,7 +31,7 @@ final public function init() {
 	 */
 	public function site_visibility_badge( $wp_admin_bar ) {
 		// Early exit if LYS feature is disabled.
-		if ( ! Features::is_enabled( 'launch-your-store' ) ) {
+		if ( ! FeaturesUtil::feature_is_enabled( 'site_visibility_badge' ) ) {
 			return;
```

Comment:
We plan to remove the main feature flag in 2025. (see Disabling the feature in the [intro post](https://developer.woocommerce.com/2024/06/18/introducing-coming-soon-mode/). 

I'm thinking we would rather not get into a scenario where users have used the main feature flag to disable the badge specifically, and then we can't easily remove the main flag from core. Therefore I've removed the main feature flag check here and replaced it with the new site_visibility_badge ff check.